src/pheval_exomiser/post_process/post_process_results_format.py

```diff
@@ -114,81 +23,131 @@ def read_exomiser_json_result(exomiser_result_path: Path) -> dict:
     return exomiser_result
 
 
-class StandardiseExomiserResult:
-    """Standardise Exomiser output into simplified gene and variant results for analysis."""
-
+class PhEvalGeneResultFromExomiserJsonCreator:
     def __init__(self, exomiser_json_result: [dict], ranking_method: str):
```

Comment:
would `ranking_method` be better named `score_name`? e.g. the value toud be `combinedScore` , `phenotypeScore`, or `pValue`